sdk/search/Azure.Search.Documents/src/AzureSearchDocumentsEventSource.cs

```diff
@@ -15,35 +18,94 @@ internal sealed class AzureSearchDocumentsEventSource : EventSource
         internal const int BatchSubmittedEvent = 2;
         internal const int RetryQueueResizedEvent = 3;
         internal const int PendingQueueResizedEvent = 4;
+        internal const int PublisherDisposedWithPendingActionsEvent = 5;
```

Comment:
We should not use the word Publisher anywhere.  That's an internal implementation detail.  They'll see this same thing by not properly disposing a SearchIndexingBufferedSender.